tendrl/commons/utils/generate_ssh.py

```diff
@@ -0,0 +1,47 @@
+from ansible_module_runner import AnsibleExecutableGenerationFailed
+from ansible_module_runner import AnsibleRunner
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+
+ANSIBLE_MODULE_PATH = "core/system/user.py"
```

Comment:
@brainfunked yes i tried, if the user is exist it will create only ssh-key, if the user is not exist it will create user with ssh-key